models/getAllTopicsModel.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../db/connection.js')
+
+const getAllTopics = () => {
+    return db.query(`
+    SELECT * FROM topics
+    ORDER BY slug ASC;
+    ;
+    `).then((result) => {
+        const data = result.rows
+        if (!data) {
```

Comment:
This error handling can be moved out and deal with in our app.js paths and error handling file. We just need to ensure we have the catch in our controller :)